lib/apply-extends.js

```diff
@@ -0,0 +1,19 @@
+var fs = require('fs')
+var path = require('path')
+var assign = require('./assign')
+
+function applyExtends (config, cwd) {
+  var defaultConfig = {}
+
+  if (config.hasOwnProperty('extends')) {
+    var pathToDefault = path.join(cwd, config.extends)
```

Comment:
I feel yargs is too general of a tool to make assumptions for shorthand extends like `my-lib`. For more context, my use case in nyc probably won't even extend a root `.nycrc` in another package. Rather, it will extend something like `my-lib/configs/.nycrc`.